downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -324,6 +324,13 @@ All the connectors listed below have been released under new major versions, due
 - Enhanced integration with Twilio services, now fully supporting all features offered through the Twilio Basic API version 2010-04-01.
 - This comprehensive update includes capabilities across messaging, voice calls, media services, and more, ensuring full utilization of Twilio's extensive functionality.
 
+#### `salesforce` package
+
+- Upgraded connector APIs with remote function syntax, along with improved documentation and examples.
```

Comment:
Should this be
```suggestion
- Upgraded connector APIs with resource function syntax, along with improved documentation and examples.
```
?

Reply:
No salesforce uses remote function. But I am not sure if we should mention that. @aashikam Shall we just mention doc and examples update and move this point to the last in line.